vendor.conf

```diff
@@ -38,10 +38,11 @@ github.com/prometheus/client_golang                 c42bebe5a5cddfc6b28cd6391033
 github.com/prometheus/client_model                  d1d2010b5beead3fa1c5f271a5cf626e40b3ad6e # v0.1.0
 github.com/prometheus/common                        287d3e634a1e550c9e463dd7e5a75a422c614505 # v0.7.0
 github.com/prometheus/procfs                        6d489fc7f1d9cd890a250f3ea3431b1744b9623f # v0.0.8
-github.com/russross/blackfriday                     05f3235734ad95d0016f6a23902f06461fcf567a # v1.5.2
+github.com/russross/blackfriday/v2                  d3b5b032dc8e8927d31a5071b56e14c89f045135 # v2.0.1
+github.com/shurcooL/sanitized_anchor_name           7bfe4c7ecddb3666a94b053b422cdd8f5aaa3615 # v1.0.0
 github.com/sirupsen/logrus                          60c74ad9be0d874af0ab0daef6ab07c5c5911f0d # v1.6.0
 github.com/syndtr/gocapability                      d98352740cb2c55f81556b63d4a1ec64c5a319c2
-github.com/urfave/cli                               bfe2e925cfb6d44b40ad3a779165ea7e8aff9212 # v1.22.0
+github.com/urfave/cli                               053ba9dd02d2bfb768da2c1c50370c59d41dc04b # v1.22.4
```

Comment:
Should be <= v1.22.1: https://github.com/opencontainers/runc/blob/2b31437caa905b7b944a891aee613e7dd0a1f898/go.mod#L22

Reply:
We need to get somebody to fix https://github.com/urfave/cli/issues/1092

Reply:
Ah, dang. I read that issue, but forgot about it

Reply:
was trying to get the other dependencies updated to the `v2` variants to be in line with moby and docker cli 😅